code/game/objects/items/weapons/shields.dm

```diff
@@ -35,6 +35,19 @@
 	icon_state = "roman_shield"
 	item_state = "roman_shield"
 
+/obj/item/weapon/shield/riot/buckler
+	name = "wooden buckler"
+	desc = "A medieval wooden buckler."
+	icon_state = "buckler"
+	item_state = "buckler"
+	materials = list()
+
+/obj/item/weapon/shield/riot/buckler/IsShield()
+	if(prob(60))
+		return 1
+	else
+		return 0
+

```

Comment:
Somehow, this proc seems incredibly humorous to me.
"Are you a shield?"
"No."
"How about now?"